src/basic/socket-util.c

```diff
@@ -900,6 +900,29 @@ bool ifname_valid(const char *p) {
         return true;
 }
 
+bool iflabel_valid(const char *p) {
+
+        if (isempty(p))
+                return false;
+
+        if (strlen(p) >= IFNAMSIZ)
+                return false;
+
+        if (dot_or_dot_dot(p))
+                return false;
+
+        while (*p) {
+                if ((unsigned char) *p >= 127U)
+                        return false;
+
+                if ((unsigned char) *p <= 32U)
```

Comment:
`ip address` command thinks space is a garbage.
```
sudo  ip address add 192.168.99.37/24 brd + dev wlp3s0 label wlp3s0 11
Error: either "local" is duplicate, or "11" is a garbage.
```
yes renamed to `address_label_valid()`